index.html

```diff
@@ -0,0 +1,350 @@
+    </header>
+    <main>
+      <article id="services" class="services-container">
+        <h2 class="section-header white-color white-line">SERVICES</h2>
+        <p class="services-text">Contrary to popular belief, Lorem Ipsum is not simply random text. It has roots in a piece of classical Latin literature from 45 BC, making it over 2000 years old. Richard McClintock, a Latin professor at Hampden-Sydney College in Virginia.</p>
+        <p class="services-text small-paragraph">Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book.</p>
+        <p class="services-text small-paragraph">Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book.</p>
+        <div class="services-category-container">
+          <section class="services-category">
+            <h3 class="services-header white-color" data-title="ux-design">UI &amp; UX DESIGN</h3>
+            <p class="services-text">There are many variations of passages of Lorem Ipsum available, but the majority have suffered alteration in some form, by injected support by humour.</p>
+          </section>
+          <section class="services-category">
+            <h3 class="services-header white-color" data-title="laboratory">LABORATORY</h3>
+            <p class="services-text">There are many variations of passages of Lorem Ipsum available, but the majority have suffered alteration in some form, by injected support by humour.</p>
+          </section>
+          <section class="services-category">
+            <h3 class="services-header white-color" data-title="development">DEVELOPMENT</h3>
+            <p class="services-text">There are many variations of passages of Lorem Ipsum available, but the majority have suffered alteration in some form, by injected support by humour.</p>
+          </section>
+        </div>
+        <section class="pricing">
+          <h3 class="sub-section-header white-color white-line">PRICING TABLES</h3>
+          <section class="pricing-tables">
+            <h4 class="pricing-header white-color">BASIC</h4>
+            <ul>
+              <li class="pricing-plan"><span class="pricing-cost">$5.99</span> /MONTH</li>
+              <li class="pricing-stats">3 Projects</li>
+              <li class="pricing-stats">2 Users</li>
+              <li class="pricing-stats">1 Gb</li>
+              <li class="pricing-stats">1 Calendar</li>
+            </ul>
+            <div class="pricing-button-container">
+              <a href=""><button class="pricing-button">SIGN UP</button></a>
```

Comment:
This is invalid markup. For something like this, the `a` element is correct (the user is going somewhere as a result of clicking on the element), but you should not use the `button` element to produce button-esque styles. Better would be to have a class of `.button` or `.action` that can be shared by both elements.